post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+:context: post-install-vsphere-zones-regions-configuration
+[id="post-install-vsphere-zones-regions-configuration"]
+=  Multiple regions and zones configuration for a cluster on vSphere
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As an administrator, you can specify multiple regions and zones for your {product-title} cluster that runs on a VMware vSphere instance. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+A failure domain describes a unique topology. A failure domain configuration lists parameters that create a topology. The following list states some of these parameters:
```

Comment:
Consider deleting the first sentence. The paragraph woks fine without it.

Reply:
Removed